test/test_samples.rb

```diff
@@ -17,6 +17,7 @@ def test_samples_osw
     all_results_output = []
     [['project_testing', 100], ['project_national', 3000]].each_with_index do |scenario, i|
       project_dir, num_samples = scenario
+      next if num_samples == 0
```

Comment:
@aspeake This is just like a convenience thing?

Reply:
Pretty much, it prevents an error from being thrown if one of the projects is set to 0